src/test/java/subway/LineAcceptanceTest.java

```diff
@@ -0,0 +1,228 @@
+                        .when().get("/lines")
+                        .then().log().all()
+                        .extract().jsonPath().getList("name", String.class);
+        //then
+        assertThat(lineNames).contains("신분당선", "분당선");
+    }
+
+
+    /**
+     * Given: 특정 지하철 노선이 등록되어 있고,
+     * When: 관리자가 해당 노선을 조회하면,
+     * Then: 해당 노선의 정보가 반환된다.
+     */
+    @DisplayName("지하철 노선을 조회한다.")
+    @Test
+    void showLine() {
+        // given
+        LineRequest lineRequestRed = new LineRequest();
+        lineRequestRed.setName("신분당선");
+        lineRequestRed.setColor("bg-red-600");
+        lineRequestRed.setUpStationId(1);
+        lineRequestRed.setDownStationId(2);
+        lineRequestRed.setDistance(10);
+
+        assertThat(createLine(lineRequestRed).statusCode()).isEqualTo(HttpStatus.CREATED.value());
+        // when
+        List<String> lineNames =
+                given().log().all()
+                        .when().get("/lines")
+                        .then().log().all()
+                        .extract().jsonPath().getList("name", String.class);
+        //then
+        assertThat(lineNames).contains("신분당선");
+    }
+
+    /**
+     * Given: 특정 지하철 노선이 등록되어 있고,
+     * When: 관리자가 해당 노선을 수정하면,
+     * Then: 해당 노선의 정보가 수정된다.
+     */
+    @DisplayName("지하철 노선을 수정한다.")
+    @Test
+    void editLine() {
+        // given
+        LineRequest lineRequestRed = new LineRequest();
+        lineRequestRed.setName("신분당선");
+        lineRequestRed.setColor("bg-red-600");
+        lineRequestRed.setUpStationId(1);
+        lineRequestRed.setDownStationId(2);
+        lineRequestRed.setDistance(10);
+
+        String lineId  =   createLine(lineRequestRed).getHeaders().get("Location").toString().split("/")[2];
+
+
+        //when
+        Map<String, String> modifyParam = new HashMap<>();
+        modifyParam.put("name", "다른분당선");
+        modifyParam.put("color", "bg-red-600");
+
+        ExtractableResponse<Response> response =
+                given().log().all()
+                        .body(modifyParam)
+                        .contentType(MediaType.APPLICATION_JSON_VALUE)
+                        .when().put("/lines/{id}", lineId)
+                        .then().log().all()
+                        .extract();
+        //then
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+
+    }
+
+    /**
+     * Given: 특정 지하철 노선이 등록되어 있고,
+     * When: 관리자가 해당 노선을 삭제하면,
+     * Then: 해당 노선이 삭제되고 노선 목록에서 제외된다.
+     */
+    @DisplayName("지하철 노선을 삭제한다.")
+    @Test
+    void deleteLine() {
+        // given
+        LineRequest lineRequestRed = new LineRequest();
+        lineRequestRed.setName("신분당선");
+        lineRequestRed.setColor("bg-red-600");
+        lineRequestRed.setUpStationId(1);
+        lineRequestRed.setDownStationId(2);
+        lineRequestRed.setDistance(10);
+
+        String lineId  =   createLine(lineRequestRed).getHeaders().get("Location").toString().split("/")[2];
```

Comment:
```suggestion
        String lineId = createLine(lineRequestRed).getHeaders().get("Location").toString().split("/")[2];
```

👀 
사소한 컨벤션이지만 가능하면 일관성을 지켜주세요

Reply:
넵~